Model/Paypal.php

```diff
@@ -421,7 +421,7 @@ protected function _buildCheckoutRequest($payment, $amount)
         $request['MerchantId'] = $this->reachHelper->getMerchantId();
         $request['ReferenceId'] = $order->getIncrementId();
         $request['ConsumerCurrency']= $order->getOrderCurrencyCode();
-        $order->getOrderCurrencyCode();
+        $order->getOrderCurrencyCode(); //What it is for?
```

Comment:
do we also want to remove the redundant call here?

Reply:
Just did it Lizzy 7784476775073aa1e7c6532a4c616f046ea40967
Thanks.